src/PhysicsSettings.cpp

```diff
@@ -18,8 +18,10 @@ PhysicsSettings::PhysicsSettings()
     }
 })
 , vehicle_mesh_asset_path("assets/models/WienerCarModel.obj")
-, vehicle_mass(1000.f)
+, vehicle_mass(500.f)
+, wheel_mass(20.f)
```

Comment:
With the new physics settings, it's much easier for the car to start tilting when turning, and you tend to tumble over when the car is driving over a mound and try to turn. I think we might want to tweak the steering values a bit after this.

Also, with the new vehicle mass we might want to adjust the bouncing force after a collision to be weaker.

Reply:
Yea it's complicated, since the mass of the car was used in a few places, like to calculate moment of inertia and stuff like that too. 

```
vehicle->mWheelsSimData.setChassisMass(settings_.vehicle_mass);
```

And this line that was used in the hot reloading, was never actually called when creating the car, so I added that in. But because reloading would mean like remaking the cars and stuff, it was too difficult,